react-components/src/data-providers/core-dm-provider/filterNodesByMappedTo3d.ts

```diff
@@ -128,16 +132,17 @@ function getRelevantObject3ds(
   connectionData: QueryResult<
     ReturnType<typeof createCheck3dConnectedEquipmentQuery>,
     SelectSourcesType
-  >
+  >,
+  includeIndirectRelations: boolean
```

Comment:
I think I would prefer 2 separate functions for `getRelevantObject3ds`. One for with indirect and one for no indirect nodes. The connectiondata type is currently wrong as non indirect query does not return non indirect clauses

Reply:
I feel like this might be a footgun at some point, as the indirect properties does not exist. I will however let you take the final decision

Reply:
It definitely exists